diff --git a/./hints/hint.py b/./hints/hints.py
index 19d3957..102d05d 100644
--- a/./hints/hint.py
+++ b/./hints/hints.py
@@ -251,6 +251,7 @@ def get_window_system() -> Type[WindowSystem]:
     :return: The window system for the current system.
     """
     window_system_type = get_window_system_type()
+    print(window_system_type)
 
     # add new waland wms here, then add a match case below to import the class
     supported_wayland_wms = {"sway", "Hyprland", "plasmashell"}
@@ -273,6 +274,8 @@ def get_window_system() -> Type[WindowSystem]:
             .strip()
         )
 
+        print(wayland_wm)
+
         match wayland_wm:
             case "sway":
                 from hints.window_systems.sway import Sway as window_system
@@ -283,6 +286,7 @@ def get_window_system() -> Type[WindowSystem]:
                 from hints.window_systems.plasmashell import \
                     Plasmashell as window_system
 
+    print(window_system)
     if not window_system:
         # adding x11 for an acurate report of the supported window systems
         supported_wayland_wms.add("x11")
